src/charm.py

```diff
@@ -30,7 +30,8 @@ def _validated_address(address: str) -> str:
 
     parsed = urlparse(address)
     if not parsed.netloc or any([parsed.scheme, parsed.path, parsed.params, parsed.query]):
-        logger.error("Invalid address (should only include netloc): %s", address)
+        logger.error("Invalid address : %s", address)
+        logger.error("Valid address must be in host:port format")
```

Comment:
Targets must be specified in host:port format

Reply:
done